vispy/scene/widgets/widget.py

```diff
@@ -5,6 +5,7 @@
 from __future__ import division
 
 import numpy as np
+from vispy.visuals import _BorderVisual
```

Comment:
The thought was that `BorderVisual` would be used by some of vispy's visuals, but that no one "externally" would really need it, so it wouldn't show up in Intellisense / autocompletion.